CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+
+  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
+  - Include a descriptive `<title>` element for accessibility. For example, `<title>GitHub icon</title>`.
+  - Add `aria-labelledby="title"` to the `<svg>` element to improve screen reader support.
```

Comment:
You're missing adding `role="img"` if you want to go all-out on accessibility (see [here](https://css-tricks.com/accessible-svgs/))